### PR TITLE
Write to console instead of file, to leverage GHA native logs

### PR DESCRIPTION
**Pull request by aldenquimby**
## Background

- I recently started using this action and noticed the run logs don't have meaningful output:
image

- To see the logs, I need to navigate back to a run summary, download an artifact, unzip, and view:
image

image

- Afterwards, I need to delete the zip + txt file from my local machine

## Proposal

- This PR proposes inlining the logs instead, so they are immediately useful and searchable within the native GHA log UI:
image

- Added benefit of removing a dependency on the upload-artifacts action

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ inputs:
     required: true
     description: 'The project id'
   branch_name:
-    required: false
+    required: true
```
Comment (aldenquimby) [inline action.yml:13]: Unrelated to the crux of the PR, it looks to me like this should be required?

Comment (duskpoet) [reply action.yml:13]: it shouldn't, Neon provides a default name that matches the id

**Review by duskpoet — CHANGES_REQUESTED** [review]
Did you have a chance to test this out? I have a feeling that bash command output is not copied to GHA output

File: action.yml (hunk 1 above)
Comment (aldenquimby) [reply action.yml]: Ok I reverted this

**Comment by aldenquimby** [comment]
@duskpoet yes I tested this by switching my project to use `neondatabase/create-branch-action@alden-test-output` and it worked. See the screenshot at the bottom of the PR description for what it looks like

**Review by duskpoet — APPROVED** [review]

**Pull request merged**